### PR TITLE
Mobil changes

### PR DESCRIPTION
**Pull request by nadjak77**
Pull Request for Issue # .

### Summary of Changes

some changes for responsive layout
fix some broken buttons
change color https://github.com/joomla/backend-template/issues/411

### CODE REVIEW

**Comment by brianteeman** [comment]
For the joomla-toolbar-button to work you need to add
`HTMLHelper::_('webcomponent', 'system/joomla-toolbar-button.min.js', ['version' => 'auto', 'relative' => true]);`

look at `layouts\joomla\toolbar\standard.php` for an example

**Comment by brianteeman** [comment]
scss files are formatted using two spaces - not a tab
You can see that in the rest of the files that you edited

**Comment by nadjak77** [comment]
I corrected the tabs to spaces in the two scss files I changed

File: administrator/templates/atum/scss/blocks/_toolbar.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@ joomla-toolbar-button {
   display:block;
 }
 .subhead {
-  position: sticky;
+  position: sticky;s
```
Comment (brianteeman) [inline administrator/templates/atum/scss/blocks/_toolbar.scss]: oops

**Comment by nadjak77** [comment]
I hope I added the HTMLHelper for the joomla-toolbar-buttons the right way

**Pull request merged**